lib/actions/global.ts

```diff
@@ -1,9 +1,8 @@
-import Big from 'big.js';
-import {BTCAmount} from '../monetary';
+import {BTCAmount, USDTAmount} from '../monetary';
 
 export interface GlobalActions {
-  totalLiquidity(): Promise<Big>; // call treasury contract, TODO: decide on the type, maybe MonetaryAmount<USDT>?
-  totalFeesEarned(): Promise<Big>; // TODO: discuss design
+  totalLiquidity(): Promise<USDTAmount>; // call treasury contract
```

Comment:
There is a bit of a tricky thing here: some of the collateral will be in the Uniswap pools for bootstrapping. So technically we have to add all liquidity from the treasury plus the liquidity from all the Uniswap pools.
On second thought though, using part of the collateral to bootstrap the pools is an optimization that makes our lives here and at some other points a bit harder. So we might switch to just having the full collateral in the treasury. @danhper @gregdhill what do you guys think?